.rubocop.yml

```diff
@@ -15,42 +15,42 @@ AllCops:
 
 # Using instance variables might make tests fail based on their order.
 RSpec/InstanceVariable:
-   AssignmentOnly: true
-   Enabled: true
+  AssignmentOnly: true
```

Comment:
I assume these changes are all wanted by the linter?

Reply:
No idea but i don't think there were any changes made to that file in this commit.